docker-compose.yml

```diff
@@ -0,0 +1,89 @@
+version: '3.7'
+
+networks:
+  nlpjs:
+
+services:
+  mongo:
+    image: mongo:4.0.11
+    container_name: nlpjs_mongo
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=myadmin
+      - MONGO_INITDB_ROOT_PASSWORD=myadmin
+      - MONGO_INITDB_DATABASE=nlpjs
+    volumes:
+      - ${PWD}/mongo_data/data:/data/db
+      - ${PWD}/mongo_data/init:/docker-entrypoint-initdb.d/
+      # - ${PWD}/mongo_data/configdb:/data/configdb
+    expose:
+      - 27017
+    networks:
+      - nlpjs
+    healthcheck:
+      test: ["CMD", "mongostat", "-u", "myadmin", "-p", "myadmin", "--authenticationDatabase=admin", "--json", "-n", "1"]
+      # test: ["CMD", "mongotop", "-u", "myadmin", "-p", "myadmin", "--authenticationDatabase=admin", "--json", "-n", "1"]
```

Comment:
Remove it or add a comment explaining how it contributes

Reply:
I don't clean the variant, I have explain it.